docker-compose.yml

```diff
@@ -285,10 +285,10 @@ services:
     tty: true
     volumes:
       - .:/app
-      - bundle-jruby-9.2-latest:/usr/local/bundle
+      - bundle-jruby-9.2:/usr/local/bundle
```

Comment:
I believe you need to also change the `volumes:` section at the bottom of the docker-compose...?

Reply:
Correct! I always forget about these.

Reply:
Fixed!